Sources/Plasma/FeatureLib/pfPython/cyPythonInterface.cpp

```diff
@@ -784,14 +800,33 @@ PYTHON_METHOD_DEFINITION(ptErrorRedirector, excepthook, args)
     if (!PyArg_ParseTuple(args, "OOO", &exc, &value, &tb))
         PYTHON_RETURN_ERROR;
 
-    self->fThis->ExceptHook(exc, value, tb);
+    self->fThis->UncaughtHook(exc, value, tb);
+
+    PYTHON_RETURN_NONE;
+}
+
+PYTHON_METHOD_DEFINITION(ptErrorRedirector, unraisablehook, args)
+{
+    PyObject* exc;
+    if (!PyArg_ParseTuple(args, "O", &exc))
+        PYTHON_RETURN_ERROR;
+
+    self->fThis->UnraisableHook(exc);
 
     PYTHON_RETURN_NONE;
 }
 
+PYTHON_METHOD_DEFINITION_NOARGS(ptErrorRedirector, flush)
+{
+    // no-op
+    PYTHON_RETURN_NONE;
+}
+
 PYTHON_START_METHODS_TABLE(ptErrorRedirector)
     PYTHON_METHOD(ptErrorRedirector, write, "Adds text to the output object"),
-    PYTHON_METHOD(ptErrorRedirector, excepthook, "Handles exceptions"),
+    PYTHON_METHOD(ptErrorRedirector, excepthook, "Handles uncaught exceptions"),
```

Comment:
Keeping this as `excepthook` for compatibility?

Reply:
It's still called `sys.excepthook`